sdcm/utils/nemesis.py

```diff
@@ -0,0 +1,80 @@
+from functools import cached_property
+from pathlib import Path
+
+import yaml
+from jinja2 import Template
+
+
+BASE_JOB_NAME = "longevity-10gb-3h"
+
+
+class NemesisJobGenerator:
+
+    CONFIG_TEMPLATE_NAME = f"template-{BASE_JOB_NAME}.yaml.j2"
+    JOB_TEMPLATE_NAME = f"template-{BASE_JOB_NAME}.jenkinsfile.j2"
```

Comment:
If we want to have the template of the same job on Azure / GCE backend, maybe would be easier to set with the same template, but just pick the backend.